apps/judicial-system/web/messages/Core/errors.ts

```diff
@@ -115,4 +115,10 @@ export const errors = defineMessages({
     defaultMessage: 'Upp kom villa við að eyða ákærulið',
     description: 'Notaður sem villuskilaboð þegar ekki gengur að eyða ákærulíð',
   },
+  getCaseToOpenError: {
```

Comment:
Consider renaming to remove "Error" from the end to be in line with the other error string titles